test/test_ocp_s2i_local_templates.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from container_ci_suite.openshift import OpenShiftAPI
+
+from conftest import VARS
+
+
+DEPLOYED_PGSQL_IMAGE = "quay.io/sclorg/postgresql-12-c8s"
```

Comment:
We deploy, presumably, containers on different distros with different versions, but this is hardcoded to c8s?

TODO